doc/rst/users-guide/base/vars.rst

```diff
@@ -123,11 +123,11 @@ linefeeds in this example is simply for readability and has no
 practical impact).
 
 The rule of thumb is that any variable which has neither a type
-specification nor an initializer will share the type and/or
+specification nor an initial value will share the type and/or
 initializer expressions of the next variable which has one or both.
```

Comment:
Thanks for noticing this, it looks like a good change to me.  This last occurrence on line 127 seems like it could be changed to "initial value" as well?  Other than that, I think this is ready to go.